### ISSUE TITLE
Segfault when TURN server is added as a candidate

### ISSUE THREAD

**Issue by creikey** [godotengine/webrtc-native#20]
Stack:
```Connecting to ws://localhost:4876
Connected 495950498
Peer connected 1, error
(opensslidentity.cc:45): Making key pair
(peerconnection.cc:849): Session: 2587601506333643374 is destroyed.
(opensslidentity.cc:93): Returning key pair
(opensslcertificate.cc:58): Making certificate for WebRTC
(opensslcertificate.cc:105): Returning certificate
(peerconnection.cc:849): Session:  is destroyed.
ERROR: PeerConnection could not be created
ERROR: peer_connection.get() == nullptr
ERROR: add_peer: Condition "p_peer->get_connection_state() != WebRTCPeerConnection::STATE_NEW" is true. Returned: ERR_INVALID_PARAMETER
   At: modules/webrtc/webrtc_multiplayer.cpp:242.
handle_crash: Program crashed with signal 11
Dumping the backtrace. Please include this when reporting the bug on https://github.com/godotengine/godot/issues
[1] /usr/lib/libc.so.6(+0x3c3e0) [0x7fafbc5753e0] (??:0)
[2] godot_webrtc::WebRTCLibPeerConnection::close() (??:0)
[3] godot_webrtc::WebRTCLibPeerConnection::~WebRTCLibPeerConnection() (??:0)
[4] void godot::_godot_class_destroy_func<godot_webrtc::WebRTCLibPeerConnection>(void*, void*, void*) (??:0)
[5] godot() [0x26016ed] (??:?)
[6] godot() [0xc6af00] (??:0)
[7] godot() [0x291f269] (??:?)
[8] godot() [0xa0d558] (??:0)
[9] godot() [0xbf8599] (??:0)
[10] godot() [0xb385d6] (??:0)
[11] godot() [0x2a0b35b] (??:?)
[12] godot() [0x2a29818] (??:?)
[13] godot() [0xc4e60f] (??:0)
[14] godot() [0xc57fc6] (??:0)
[15] godot() [0xc58ed7] (??:0)
[16] godot() [0xbd8eff] (??:0)
[17] godot() [0xc4e6d3] (??:0)
[18] godot() [0xb38596] (??:0)
[19] godot() [0x2a0b35b] (??:?)
[20] godot() [0x2a29818] (??:?)
[21] godot() [0xc4e60f] (??:0)
[22] godot() [0xc57fc6] (??:0)
[23] godot() [0xc58fc0] (??:0)
[24] godot() [0x278631e] (??:?)
[25] godot() [0x26fa0f6] (??:?)
[26] godot() [0x2786799] (??:?)
[27] godot() [0x28ea179] (??:?)
[28] godot() [0xc4e6d3] (??:0)
[29] godot() [0xb38596] (??:0)
[30] godot() [0x2a0b35b] (??:?)
[31] godot() [0x2a29740] (??:?)
[32] godot() [0x18c4a6f] (??:0)
[33] godot() [0xbd90b4] (??:0)
[34] godot() [0x186bb0e] (??:0)
[35] godot() [0x18a419e] (??:0)
[36] godot() [0x2949ee0] (??:?)
[37] godot() [0x8582dd] (??:0)
[38] /usr/lib/libc.so.6(__libc_start_main+0xf2) [0x7fafbc560002] (??:0)
[39] godot() [0x86695e] (??:0)
-- END OF BACKTRACE --
```

**Comment by Faless** [comment]
Duplicate of #7 likely related to wrong configuration dictionary (can't know for sure since no minimal reproduction project is provided), the `::close` crash is fixed in #11 (unreleased yet.)

**Comment by creikey** [comment]
Yes, this fixed the issue. Thanks for linking as duplicate.
